tests/unit_tests/simtel/test_simtel_config_writer.py

```diff
@@ -38,6 +38,12 @@ def test_write_array_config_file(
     )
     assert file_has_text(file, "TELESCOPE == 1")
 
+    # simtel configuration files need to end with two new lines
+    with open(file) as f:
+        lines = f.readlines()
+        assert lines[-2].endswith("#endif \n")
```

Comment:
I would actually just check for `\n` instead of the exact `#endif \n` because in the future we might not have/need the `#endif`. 
I added a commit to that effect, if you agree, please merge, otherwise revert and merge.

Reply:
Good, thanks!